CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog for AWS IoT Device Shadow library
-## v1.1.1 (May 2021)
+
+## v1.1.1 (July 2021)
 
 ### Other
  - [#85](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/pull/85) Fix CBMC proof for `Shadow_AssembleTopicString` API.
+ - [#86](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/pull/86) Fix description of CBMC in README
+ - [#87](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/pull/87) Upgrade proof tools: Litani 1.10.0 and starter kit
```

Comment:
This is not necessarily a user-facing update as it is just an update in our CBMC proof infrastructure. So, we can remove that